src/js/filter.ts

```diff
@@ -1,19 +1,56 @@
 /* global document, window */
 
+import type Choices from "choices.js";
 import type { CircleMarker, FeatureGroup } from "leaflet";
 import type { CityId, CityEntry } from "./types";
 
+/**
+ * Return true if the city should be rendered on the map.
+ *
+ * Note that search takes priority. If certain cities are selected via
+ * search, they should be shown regardless of the filters.
+ */
+const shouldBeSelected = (
```

Comment:
Maybe shouldBeRendered is a better name? 
Since "selected" as a verb is being used for filter options.

Reply:
Excellent suggestion!